### ISSUE TITLE
update to jetpack 4.3

### ISSUE THREAD

**Issue by eliabruni** [idavis/jetson-containers#26]
Great work!
Is there any plan to upgrade jetson containers to jetpack 4.3? 
I tried the copy/paste approach but run into problems.

When I run:

```
make l4t-32.3.1-jax
```

I get:

```
...
Start L4T BSP package installation
QEMU binary is not available, looking for QEMU from host system
Found /usr/bin/qemu-aarch64-static
Installing QEMU binary in rootfs
/usr/bin/install: '/usr/bin/qemu-aarch64-static' and '//usr/bin/qemu-aarch64-static' are the same file
The command '/bin/sh -c apt-get update && apt-get install -y --no-install-recommends     bzip2     lbzip2     sudo     &&     echo "667462228a705bf3a64e83d0731c192a *./${DRIVER_PACK}" | md5sum -c - &&     tar -xpj --overwrite -f ./${DRIVER_PACK} &&     sed -i '/.*tar -I lbzip2 -xpmf ${LDK_NV_TEGRA_DIR}\/config\.tbz2.*/c\tar -I lbzip2 -xpm --overwrite -f ${LDK_NV_TEGRA_DIR}\/config.tbz2' ./Linux_for_Tegra/apply_binaries.sh &&     ./Linux_for_Tegra/apply_binaries.sh -r / &&     rm -rf ./Linux_for_Tegra &&     rm ./${DRIVER_PACK}     &&     apt-get purge --autoremove -y bzip2 lbzip2 &&     apt-get clean &&     rm -rf /var/lib/apt/lists/*' returned a non-zero code: 1
Makefile:13: recipe for target '32.3.1-jax' failed
make[1]: *** [32.3.1-jax] Error 1
make[1]: Leaving directory '/home/elia/aigritec/code/xavier-containers/docker/l4t'
Makefile:42: recipe for target 'l4t-32.3.1-jax' failed
make: *** [l4t-32.3.1-jax] Error 2
```

I thoroughly checked all my changes, which basically reflect the changes done in #23, but I cannot find any hints. I added changes for jax only, but that shouldn't make a difference.

I hope I can get some help. Having jetson-containers updated to 4.3 would be really nice I think, especially because it could then exploit Over the Air Updates for future releases, via APT GET NVIDIA repo.

**Comment by eliabruni** [comment]
If helpful, I could provide the code to reproduce the error via a pull request.

**Comment by idavis** [comment]
Hi @eliabruni, 

Sorry for the delay, I missed the notification. I've created #27 to address 4.3. It takes at least an entire day to add a new version of JetPack and careful analysis of the manifests. I'm hoping to make this a lot easier. Are you able to build other versions of JetPack using the current version of the repository? If not, take a look at my blog post, [Jetson Container - Introduction](https://codepyre.com/2019/07/jetson-containers-introduction/) which may help. If you don't want to deal with copying QEMU into containers, take a look at [ARMing Yourself](https://codepyre.com/2019/12/arming-yourself/).

You can also break that line into multiple `RUN` commands and see where it breaks. Since I `--squash` all base images, the optimization of a single `RUN` command isn't really needed now.

**Comment by idavis** [comment]
@eliabruni,

The issue you are seeing is due to the OTA key registration that is being done. They are doing silly things in their ARM checks so that they can `chroot`. Working on a workaround.

**Comment by eliabruni** [comment]
Hi @idavis, great you're looking into this, I honestly wouldn't have been able to figure it out. #27  looks amazing! I will just wait for it to be done then. 
And of course, I looked into your [ Jetson Container - Introduction](https://codepyre.com/2019/07/jetson-containers-introduction), as like as all other Jetson-related posts. They are being super helpful for going into production with a Jetson product! 

I am not sure this is the right place, but I'll say it anyway: IMO it would be extremely useful to have a post about security with Jetson (especially within containerized workloads; e.g., is it SIngularity a good way?). Hope you don't mind receiving a post request:)

**Comment by idavis** [comment]
@eliabruni Closing as I've merged #27.

**Comment by eliabruni** [comment]
Thank you, @idavis, much appreciated! I'll try it out tomorrow the latest.
